Moonshot/player/PlayerAudio.gd

```diff
@@ -0,0 +1,105 @@
+extends Node
```

Comment:
Would it make more sense to have sound effects managed by an autoloader? I'm not sure just wanted to know if there was a reason that you chose to do it as a child of player.